tests/java/verifiers/src/main/java/io/vectorized/compaction/App.java

```diff
@@ -16,13 +16,15 @@ public static class Metrics {
     public long min_writes = 0;
```

Comment:
Could you please add some overview about code test changes in commit message

I am not sure that I clearly understand changes

Reply:
done

Reply:
it's a workaround for https://github.com/redpanda-data/redpanda/pull/8624#issuecomment-1421621532 dormant issue hidden by the bug which is being fixed

Reply:
It is a disabled test, why workaround? Isn't it easier if it repros consistently to be able to debug?

Reply:
Coverage we get from this test (it already found high/sev issues) outweighs the 8624 since its severity isn't obvious. It's disabled in a sense there is a workaround and not in a sense there is ok_to_fail.

Reply:
I've added a comment to the 8624 issue